frontend/src/libs/components/scroll-to-top-button/scroll-to-top-button.tsx

```diff
@@ -0,0 +1,59 @@
+import { getValidClassNames } from '~/libs/helpers/helpers.js';
+import { useCallback, useEffect, useState } from '~/libs/hooks/hooks.js';
+
+import { IconButton } from '../icon-button/icon-button.js';
+import { ScrollValue } from './libs/enums.js';
+import styles from './styles.module.scss';
+
+const ScrollToTop: React.FC = () => {
+  const [hasScrollToTopButton, setHasScrollToTopButton] = useState(false);
+  const [scrollValue, setScrollValue] = useState(ScrollValue.TOP as number);
+
+  useEffect(() => {
+    window.addEventListener('scroll', () => {
+      if (window.scrollY > ScrollValue.MIN_SCROLLED_VALUE) {
+        setHasScrollToTopButton(true);
+
+        const position = window.scrollY;
+        const height =
+          document.documentElement.scrollHeight -
+          document.documentElement.clientHeight;
+        const scrollProgress = Math.round((position / height) * 100);
```

Comment:
**B31**

Reply:
Do you mean I should create for this const?
because I think it is evidently that this number is for exchanging into a percentage, isn't it?
Or it's better to add a percentage character?

Reply:
[B31. The code does not use “magic values”, each of them has a separate variable named as a constant.](https://github.com/BinaryStudioAcademy/quality-criteria/blob/production/src/javascript.md#formatting)

just move "100" to a constant

Reply:
I see, but it looks weird to me)
For curiosity: in case I have to square the number, do I need to create a constant for number 2? 
Maybe in my case, it is better to add a percentage character at the end?